core/modules/wdb/ChunkResource.cc

```diff
@@ -196,9 +206,38 @@ class Backend {
         }
         return true;
     }
+
     void discard(ScTableVector const& v) {
         _discard(v.begin(), v.end());
     }
+
+    enum LockStatus {UNLOCKED, LOCKED_OTHER, LOCKED_OURS};
+    std::string strLockStatus(LockStatus ls) {
```

Comment:
I'd like to advocate again to prefer stream insert operators (`std::ostream& operator<<(std::ostream&, LockStatus)`) over string conversion methods with random names. In that way you don't need to remember the name of the method and can do `out << ls` instead of `out << strLockStatus(ls)`. Boost format is supposed to work correctly with the types which define stream inserters.